Core/automation/lib/python/core/utils.py

```diff
@@ -19,6 +19,11 @@
 except:
     from org.eclipse.smarthome.core.thing import ChannelUID
 
+try:
+    from org.openhab.core.library.types import QuantityType, DecimalType, PercentType
+except:
+    from org.eclipse.smarthome.core.library.types import QuantityType, DecimalType, PercentType
+
```

Comment:
This isn't needed. We already have core.jsr223.scope imported, and these are in there. So just use scope.QuantityType, etc.

Reply:
Done